docs/semgrep-app/notifications.md

```diff
@@ -106,7 +106,35 @@ semgrep:
     GITLAB_TOKEN: $PAT
 ```
 
-NOTE: GitLab MR comments are only available to logged-in semgrep.dev users, requiring both a Semgrep deployment ID and a Semgrep API token.
+If you would like to allow Semgrep to automatically suggest quick fixes, visit the [projects page](https://semgrep.dev/orgs/-/projects), select a project, and toggle "Autofix" on.
+
```

Comment:
Is this now redundant with the autofix section below?

Reply:
Good catch. Removing it.